### ISSUE TITLE
Error: Could not write JSON: For input string: "0.000000"

### ISSUE THREAD

**Issue by mnsnoop** [rain0r/ampd#328]
I'm getting the following error every second. Probably something misconfigured on my part but I'm not good enough with java to know what.

`org.springframework.messaging.converter.MessageConversionException: Could not write JSON: For input string: "0.000000" (through reference chain: org.hihn.ampd.server.message.outgoing.StatePayload["serverStatus"]->org.bff.javampd.server.MPDServerStatus["mixRampDb"]); nested exception is com.fasterxml.jackson.databind.JsonMappingException: For input string: "0.000000" (through reference chain: org.hihn.ampd.server.message.outgoing.StatePayload["serverStatus"]->org.bff.javampd.server.MPDServerStatus["mixRampDb"])
        at org.springframework.messaging.converter.MappingJackson2MessageConverter.convertToInternal(MappingJackson2MessageConverter.java:274) ~[spring-messaging-5.3.20.jar!/:5.3.20]
        at org.springframework.messaging.converter.AbstractMessageConverter.toMessage(AbstractMessageConverter.java:201) ~[spring-messaging-5.3.20.jar!/:5.3.20]
        at org.springframework.messaging.converter.CompositeMessageConverter.toMessage(CompositeMessageConverter.java:96) ~[spring-messaging-5.3.20.jar!/:5.3.20]
        at org.springframework.messaging.core.AbstractMessageSendingTemplate.doConvert(AbstractMessageSendingTemplate.java:181) ~[spring-messaging-5.3.20.jar!/:5.3.20]
        at org.springframework.messaging.core.AbstractMessageSendingTemplate.convertAndSend(AbstractMessageSendingTemplate.java:150) ~[spring-messaging-5.3.20.jar!/:5.3.20]
        at org.springframework.messaging.core.AbstractMessageSendingTemplate.convertAndSend(AbstractMessageSendingTemplate.java:129) ~[spring-messaging-5.3.20.jar!/:5.3.20]
        at org.springframework.messaging.core.AbstractMessageSendingTemplate.convertAndSend(AbstractMessageSendingTemplate.java:122) ~[spring-messaging-5.3.20.jar!/:5.3.20]
        at org.hihn.ampd.server.sender.Publisher.publishState(Publisher.java:58) ~[classes!/:1.4.2]
        at jdk.internal.reflect.GeneratedMethodAccessor58.invoke(Unknown Source) ~[na:na]
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
        at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]
        at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:84) ~[spring-context-5.3.20.jar!/:5.3.20]
        at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-5.3.20.jar!/:5.3.20]
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[na:na]
        at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305) ~[na:na]
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
        at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
Caused by: com.fasterxml.jackson.databind.JsonMappingException: For input string: "0.000000" (through reference chain: org.hihn.ampd.server.message.outgoing.StatePayload["serverStatus"]->org.bff.javampd.server.MPDServerStatus["mixRampDb"])
        at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:392) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:351) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at com.fasterxml.jackson.databind.ser.std.StdSerializer.wrapAndThrow(StdSerializer.java:316) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:782) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:774) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at com.fasterxml.jackson.databind.ObjectMapper.writeValue(ObjectMapper.java:3160) ~[jackson-databind-2.13.3.jar!/:2.13.3]
        at org.springframework.messaging.converter.MappingJackson2MessageConverter.convertToInternal(MappingJackson2MessageConverter.java:256) ~[spring-messaging-5.3.20.jar!/:5.3.20]
        ... 18 common frames omitted`

**Comment by rain0r** [comment]
Hey there,

that's no misconfiguration on your part.

This is due to an issue with `javampd`, the library that `ampd` uses: https://github.com/finnyb/javampd/issues/73

This issue should be fixed in release `ampd-1.4.3`: 

https://github.com/rain0r/ampd/releases/tag/ampd-1.4.3

Please let me know, if the error still occurs.

**Comment by mnsnoop** [comment]
1.4.4 fixed that issue but I have found another. When mpd has streams in the queue ampd will not show anything in the queue. 

Here's a debug of ampd receiving the queue without a stream in it.

```
2022-06-19 19:59:28.573 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : start command: ping

2022-06-19 19:59:28.574 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : first response line is: OK
2022-06-19 19:59:28.574 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : the response was ok
2022-06-19 19:59:28.575 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : start command: playlistinfo

2022-06-19 19:59:28.576 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : first response line is: file: Full Albums/Aphex Twin/[1987] Analogue Bubblebath EP1 [1991]/01 - Analogue Bubblebath 1.mp3
2022-06-19 19:59:28.576 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : the response was ok
2022-06-19 19:59:28.576 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : file: Full Albums/Aphex Twin/[1987] Analogue Bubblebath EP1 [1991]/01 - Analogue Bubblebath 1.mp3
2022-06-19 19:59:28.577 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Last-Modified: 2007-07-17T04:12:52Z
2022-06-19 19:59:28.577 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Artist: Aphex Twin (aka AFX)
2022-06-19 19:59:28.578 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Title: Analogue Bubblebath 1
2022-06-19 19:59:28.578 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Album: Analogue Bubblebath EP1
2022-06-19 19:59:28.578 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Track: 1
2022-06-19 19:59:28.579 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Date: 1987
2022-06-19 19:59:28.579 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Genre: IDM
2022-06-19 19:59:28.579 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Time: 282
2022-06-19 19:59:28.580 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : duration: 282.395
2022-06-19 19:59:28.580 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Pos: 0
2022-06-19 19:59:28.580 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Id: 15
2022-06-19 19:59:28.581 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : file: Full Albums/Aphex Twin/[1987] Analogue Bubblebath EP1 [1991]/02 - Isopropophlex.mp3
2022-06-19 19:59:28.581 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Last-Modified: 2007-07-17T04:04:20Z
2022-06-19 19:59:28.581 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Artist: Aphex Twin (aka AFX)
2022-06-19 19:59:28.582 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Title: Isopropophlex
2022-06-19 19:59:28.582 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Album: Analogue Bubblebath EP1
2022-06-19 19:59:28.582 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Track: 2
2022-06-19 19:59:28.583 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Date: 1987
2022-06-19 19:59:28.583 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Genre: IDM
2022-06-19 19:59:28.584 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Time: 130
2022-06-19 19:59:28.584 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : duration: 130.441
2022-06-19 19:59:28.584 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Pos: 1
2022-06-19 19:59:28.585 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Id: 16
2022-06-19 19:59:28.585 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Last-Modified ->  2007-07-17T04
2022-06-19 19:59:28.586  WARN 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : Processor not found - Last-Modified: 2007-07-17T04:12:52Z
2022-06-19 19:59:28.586 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Artist ->  Aphex Twin (aka AFX)
2022-06-19 19:59:28.586 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Title ->  Analogue Bubblebath 1
2022-06-19 19:59:28.587 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Album ->  Analogue Bubblebath EP1
2022-06-19 19:59:28.587 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Track ->  1
2022-06-19 19:59:28.588 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Date ->  1987
2022-06-19 19:59:28.588 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Genre ->  IDM
2022-06-19 19:59:28.588 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Time ->  282
2022-06-19 19:59:28.589 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: duration ->  282.395
2022-06-19 19:59:28.589  WARN 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : Processor not found - duration: 282.395
2022-06-19 19:59:28.590 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Pos ->  0
2022-06-19 19:59:28.590 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Id ->  15
2022-06-19 19:59:28.591 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Last-Modified ->  2007-07-17T04
2022-06-19 19:59:28.591  WARN 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : Processor not found - Last-Modified: 2007-07-17T04:04:20Z
2022-06-19 19:59:28.591 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Artist ->  Aphex Twin (aka AFX)
2022-06-19 19:59:28.592 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Title ->  Isopropophlex
2022-06-19 19:59:28.592 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Album ->  Analogue Bubblebath EP1
2022-06-19 19:59:28.593 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Track ->  2
2022-06-19 19:59:28.593 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Date ->  1987
2022-06-19 19:59:28.593 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Genre ->  IDM
2022-06-19 19:59:28.596 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Time ->  130
2022-06-19 19:59:28.597 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: duration ->  130.441
2022-06-19 19:59:28.597  WARN 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : Processor not found - duration: 130.441
2022-06-19 19:59:28.597 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Pos ->  1
2022-06-19 19:59:28.598 DEBUG 6657 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Id ->  16
2022-06-19 19:59:28.598 DEBUG 6657 --- [MessageBroker-1] o.s.m.s.b.SimpleBrokerMessageHandler     : Processing MESSAGE destination=/topic/queue session=null payload=[{"name":"Analogue Bubblebath 1","title":"Analogue Bubblebath 1","artistName":"A...(truncated)
```

![without stream](https://user-images.githubusercontent.com/100872211/174505550-7039252b-7ff0-438e-b235-715124fad20f.png)

And this is with a stream in the queue.

```
2022-06-19 20:03:48.523 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : start command: ping

2022-06-19 20:03:48.524 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : first response line is: OK
2022-06-19 20:03:48.524 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : the response was ok
2022-06-19 20:03:48.525 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : start command: playlistinfo

2022-06-19 20:03:48.526 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : first response line is: file: Full Albums/Aphex Twin/[1987] Analogue Bubblebath EP1 [1991]/01 - Analogue Bubblebath 1.mp3
2022-06-19 20:03:48.527 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : the response was ok
2022-06-19 20:03:48.527 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : file: Full Albums/Aphex Twin/[1987] Analogue Bubblebath EP1 [1991]/01 - Analogue Bubblebath 1.mp3
2022-06-19 20:03:48.528 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Last-Modified: 2007-07-17T04:12:52Z
2022-06-19 20:03:48.528 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Artist: Aphex Twin (aka AFX)
2022-06-19 20:03:48.529 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Title: Analogue Bubblebath 1
2022-06-19 20:03:48.529 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Album: Analogue Bubblebath EP1
2022-06-19 20:03:48.530 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Track: 1
2022-06-19 20:03:48.530 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Date: 1987
2022-06-19 20:03:48.531 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Genre: IDM
2022-06-19 20:03:48.531 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Time: 282
2022-06-19 20:03:48.532 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : duration: 282.395
2022-06-19 20:03:48.532 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Pos: 0
2022-06-19 20:03:48.532 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Id: 15
2022-06-19 20:03:48.533 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : file: Full Albums/Aphex Twin/[1987] Analogue Bubblebath EP1 [1991]/02 - Isopropophlex.mp3
2022-06-19 20:03:48.533 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Last-Modified: 2007-07-17T04:04:20Z
2022-06-19 20:03:48.534 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Artist: Aphex Twin (aka AFX)
2022-06-19 20:03:48.534 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Title: Isopropophlex
2022-06-19 20:03:48.535 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Album: Analogue Bubblebath EP1
2022-06-19 20:03:48.535 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Track: 2
2022-06-19 20:03:48.536 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Date: 1987
2022-06-19 20:03:48.536 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Genre: IDM
2022-06-19 20:03:48.537 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Time: 130
2022-06-19 20:03:48.537 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : duration: 130.441
2022-06-19 20:03:48.537 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Pos: 1
2022-06-19 20:03:48.538 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Id: 16
2022-06-19 20:03:48.538 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : file: https://streamer.radio.co/s2c3cc784b/listen
2022-06-19 20:03:48.539 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Pos: 2
2022-06-19 20:03:48.539 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : Id: 17
2022-06-19 20:03:48.540 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Last-Modified ->  2007-07-17T04
2022-06-19 20:03:48.540  WARN 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : Processor not found - Last-Modified: 2007-07-17T04:12:52Z
2022-06-19 20:03:48.541 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Artist ->  Aphex Twin (aka AFX)
2022-06-19 20:03:48.541 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Title ->  Analogue Bubblebath 1
2022-06-19 20:03:48.542 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Album ->  Analogue Bubblebath EP1
2022-06-19 20:03:48.542 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Track ->  1
2022-06-19 20:03:48.543 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Date ->  1987
2022-06-19 20:03:48.543 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Genre ->  IDM
2022-06-19 20:03:48.544 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Time ->  282
2022-06-19 20:03:48.545 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: duration ->  282.395
2022-06-19 20:03:48.546  WARN 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : Processor not found - duration: 282.395
2022-06-19 20:03:48.547 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Pos ->  0
2022-06-19 20:03:48.548 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Id ->  15
2022-06-19 20:03:48.549 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Last-Modified ->  2007-07-17T04
2022-06-19 20:03:48.550  WARN 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : Processor not found - Last-Modified: 2007-07-17T04:04:20Z
2022-06-19 20:03:48.550 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Artist ->  Aphex Twin (aka AFX)
2022-06-19 20:03:48.551 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Title ->  Isopropophlex
2022-06-19 20:03:48.551 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Album ->  Analogue Bubblebath EP1
2022-06-19 20:03:48.552 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Track ->  2
2022-06-19 20:03:48.552 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Date ->  1987
2022-06-19 20:03:48.553 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Genre ->  IDM
2022-06-19 20:03:48.553 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Time ->  130
2022-06-19 20:03:48.554 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: duration ->  130.441
2022-06-19 20:03:48.554  WARN 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : Processor not found - duration: 130.441
2022-06-19 20:03:48.555 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Pos ->  1
2022-06-19 20:03:48.555 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Id ->  16
2022-06-19 20:03:48.556 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Pos ->  2
2022-06-19 20:03:48.557 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : added tag to song: Id ->  17
2022-06-19 20:03:48.557  WARN 6730 --- [MessageBroker-1] org.bff.javampd.song.MPDTagConverter     : Unable to convert tag to Integer: null
2022-06-19 20:03:48.558 DEBUG 6730 --- [MessageBroker-1] o.s.m.s.b.SimpleBrokerMessageHandler     : Processing MESSAGE destination=/topic/queue session=null payload=[{"name":"Analogue Bubblebath 1","title":"Analogue Bubblebath 1","artistName":"A...(truncated)
2022-06-19 20:03:48.559 DEBUG 6730 --- [MessageBroker-1] o.s.m.s.b.SimpleBrokerMessageHandler     : Broadcasting to 1 sessions.
2022-06-19 20:03:48.568 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : start command: ping

2022-06-19 20:03:48.569 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : first response line is: OK
2022-06-19 20:03:48.569 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : the response was ok
2022-06-19 20:03:48.570 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : start command: status

2022-06-19 20:03:48.574 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : first response line is: volume: 100
2022-06-19 20:03:48.575 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : the response was ok
2022-06-19 20:03:48.575 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : volume: 100
2022-06-19 20:03:48.576 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : repeat: 0
2022-06-19 20:03:48.576 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : random: 0
2022-06-19 20:03:48.577 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : single: 0
2022-06-19 20:03:48.577 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : consume: 0
2022-06-19 20:03:48.578 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : partition: default
2022-06-19 20:03:48.578 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : playlist: 26
2022-06-19 20:03:48.578 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : playlistlength: 3
2022-06-19 20:03:48.579 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : mixrampdb: 1.000000
2022-06-19 20:03:48.579 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : state: stop
2022-06-19 20:03:48.580  WARN 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDServerStatus   : Response did not contain status xfade:
2022-06-19 20:03:48.581 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : start command: currentsong

2022-06-19 20:03:48.582 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : first response line is: OK
2022-06-19 20:03:48.583 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : the response was ok
2022-06-19 20:03:48.583  WARN 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDServerStatus   : Response did not contain status error:
2022-06-19 20:03:48.584 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : start command: status

2022-06-19 20:03:48.585 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : first response line is: volume: 100
2022-06-19 20:03:48.586 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : the response was ok
2022-06-19 20:03:48.587 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : volume: 100
2022-06-19 20:03:48.587 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : repeat: 0
2022-06-19 20:03:48.587 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : random: 0
2022-06-19 20:03:48.588 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : single: 0
2022-06-19 20:03:48.588 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : consume: 0
2022-06-19 20:03:48.589 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : partition: default
2022-06-19 20:03:48.589 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : playlist: 26
2022-06-19 20:03:48.590 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : playlistlength: 3
2022-06-19 20:03:48.590 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : mixrampdb: 1.000000
2022-06-19 20:03:48.591 DEBUG 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDSocket         : state: stop
2022-06-19 20:03:48.591  WARN 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDServerStatus   : Response did not contain status bitrate:
2022-06-19 20:03:48.592  WARN 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDServerStatus   : Response did not contain status audio:
2022-06-19 20:03:48.593  WARN 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDServerStatus   : Response did not contain status mixrampdelay:
2022-06-19 20:03:48.593  WARN 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDServerStatus   : Response did not contain status time:
2022-06-19 20:03:48.594  WARN 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDServerStatus   : Response did not contain status time:
2022-06-19 20:03:48.594  WARN 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDServerStatus   : Response did not contain status updating_db:
2022-06-19 20:03:48.595  WARN 6730 --- [MessageBroker-1] org.bff.javampd.server.MPDServerStatus   : Response did not contain status xfade:
2022-06-19 20:03:48.596 DEBUG 6730 --- [MessageBroker-1] o.s.m.s.b.SimpleBrokerMessageHandler     : Processing MESSAGE destination=/topic/state session=null payload={"mpdModesPanelMsg":{"consume":false,"crossfade":false,"random":false,"repeat":f...(truncated)
```

![with stream](https://user-images.githubusercontent.com/100872211/174505554-5c8d08e3-faf5-4385-b36c-8ff7d9a61c9e.png)

**Comment by rain0r** [comment]
Thanks for reporting, that was indeed a bug.

This one should be fixed in release `1.4.5`.

Please give it a try: https://github.com/rain0r/ampd/releases/tag/ampd-1.4.5

**Comment by mnsnoop** [comment]
Seems good, thanks!
